static/bidder-info/relevantdigital.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://{{.Host}}.relevant-digital.com/openrtb2/auction"
+maintainer:
+  email: "support@relevant-digital.com"
```

Comment:
Sent verification email. Please reply with received on the email.

Reply:
We did not receive any verification email, can you try again?